pages.fa/common/7z.md

```diff
@@ -0,0 +1,36 @@
+# 7z
+
+> آرشیو کننده فایل با نسبت فشرده سازی بالا.
+> اطلاعات بیشتر: <https://www.7-zip.org>.
+
+- آرشیو کننده فایل یا دایرکتوری:
+
+`7z a {{مسیر فایل آرشیو.7z}} {{مسیر فایل یا دایرکتوری}}`
```

Comment:
I'm not sure about this, I see a `7.` at the start of the placeholder and `z` at the end of the line. Is this easy to read for a Persian speaker?